### ISSUE TITLE
Unable to use `django-huey` with `SqliteHuey`

### ISSUE THREAD

**Issue by fabiocaccamo** [gaiacoop/django-huey#8]
Settings:
```python
DJANGO_HUEY = {
    'default': 'xxx',
    'queues': {
        'xxx': {
            'huey_class': 'huey.SqliteHuey',
            'name': 'xxx_tasks',
            'consumer': {
                'workers': 4,
                'worker_type': 'thread',
            },
        },
        'yyy': { 
            'huey_class': 'huey.SqliteHuey',
            'name': 'yyy_tasks',
            'consumer': {
                'workers': 2,
                'worker_type': 'thread',
            },
        }
    }
}
```

Exception raised:
```bash
huey.exceptions.ConfigurationError: "redis" python module not found, cannot use Redis storage backend. Run "pip install redis" to install.
```

`redis` should be optional.

**Comment by pablop94** [comment]
Fixed in [v1.1.1](https://github.com/gaiacoop/django-huey/releases/tag/v1.1.1)

**Comment by fabiocaccamo** [comment]
@pablop94 thank you for the quick fix, debugging it more I discovered this is a `huey` bug.
